leader-election-impl/src/main/java/com/palantir/leader/proxy/AwaitingLeadershipProxy.java

```diff
@@ -103,6 +110,7 @@ public void run() {
 
     private void gainLeadership() {
         try {
+            leaderLog.info("Quiescing");
             LeadershipToken leadershipToken = leaderElectionService.blockOnBecomingLeader();
             // We are now the leader, we should create a delegate so we can service calls
```

Comment:
Probably not? The "Gained leadership for {}" log will follow straight after except in the case of an error, or if we're already shutting down. The former case is already logged, though I should add logs for the latter case – I'll push those up in my next commit for this PR.